es/contract/compiler.js

```diff
@@ -29,11 +29,26 @@ import ContractBase from './index'
 import semverSatisfies from '../utils/semver-satisfies'
 import AsyncInit from '../utils/async-init'
 import { VM_TYPE } from '../tx/builder/schema'
+import PS from 'erlps-aesophia/dist/bundle'
+
+const showTerm = PS["Data.Show"].show(PS["Erlang.Type"].showErlangTerm);
+const erlBinToErlStr = (t) => PS["Erlang.Builtins"].erlang__binary_to_list__1([t]);
+const erlStrToJSStr = (t) => PS["Erlang.Type"].fromErl(PS["Erlang.Type"].stringFromErlang)(t).value0
+const jsStrToErlStr = PS["Erlang.Type"].toErl(PS["Erlang.Type"].stringToErlang)
+const mkTuple = PS["Erlang.Type"].ErlangTuple.create;
```

Comment:
`Erlang.Type` exposes nice constructors for these

Reply:
Nope - you need to pass in the type class dictionary for the inner type.